L1Trigger/L1TGlobal/python/l1tGlobalSummary_cfi.py

```diff
@@ -0,0 +1,11 @@
+#
+import FWCore.ParameterSet.Config as cms
+
+l1tGlobalSummary = cms.EDAnalyzer("L1TGlobalSummary",
+		                  AlgInputTag = cms.InputTag("gtStage2Digis"),
+                                  ExtInputTag = cms.InputTag("gtStage2Digis"),
+		                  ## ExtInputTag = cms.InputTag("gtExtFakeProd"),
+		                  MinBx          = cms.int32(-2),
+		                  MaxBx          = cms.int32(2),
+		                  DumpRecord   = cms.bool(False) # print raw uGT record
+)
```

Comment:
Hi,

Well, I literally mean to simple REMOVE this file completely:
```descriptions.add("L1TGlobalSummary", desc);``` in the C++ plugin creates
a file ```L1TGlobalSummary_cfi.py``` which you can python load elsewhere!